packages/eslint-plugin-typescript/package.json

```diff
@@ -25,7 +25,7 @@
   },
   "dependencies": {
     "requireindex": "^1.2.0",
-    "@typescript-eslint/parser": "21.0.2"
+    "typescript-eslint-parser": "21.0.2"
```

Comment:
what is a reason of renaming package back to old name?

Reply:
It causes issues during the installation in the integration tests

Reply:
We can update all the names at once when we publish, it didn't really make sense to do a partial job of it anyway